### PR TITLE
FEATURE: toggle safe search in extension

### PR DESCRIPTION
**Pull request by Sayanc93**
@mrshu - a toggle for safe search on and safe search off for the chrome extension. This issue was raised in the firefox extension repo. If you're good with this change, I'll reflect the same on the firefox repo too :)

### CODE REVIEW

**Comment by mrshu** [comment]
@Sayanc93 This feature generally looks nice: I have just one little request/suggestion: would you mind moving the option into the `options.html` file? That way it would not be seen by pretty much anyone who will ever open the popup, which I believe might be distracting.

Otherwise it looks pretty good!

File: js/popup.js [hunk 1]
```diff
@@ -154,10 +159,14 @@ window.onload = function() {
         }
 
         var special = '';
-        if(document.getElementById('adv_meanings').checked !== true) {
+        if (document.getElementById('adv_meanings').checked !== true) {
             special = '&d=1';
         }
 
+        var safe = '&kp=1';
```
Comment (mrshu) [inline js/popup.js]: @Sayanc93 Is there a good reason for having another variable (`safe` in this case) when there already is `special` in place?

**Comment by Sayanc93** [comment]
@mrshu - Hey, I implemented the changes.

**Comment by mrshu** [comment]
Thanks @Sayanc93, this looks pretty good. I tested it quite a bit and it seems to work as advertised.

Merging!

**Pull request merged**